news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: Kyoku 2024 Results
+date: 2024-09-19 19:30:00 +0000
+---
+
+While the seasons drifted over our heads, mappers and musicians were busy competing to create the greatest beatmaps of their own creation.
+
+![](https://assets.ppy.sh/contests/211/banner.jpg)
+
+If you don't recall, in [Kyoku](/wiki/Contests/Kyoku/2024) was a contest where teams of mappers and music producers needed to work together and collaborate on creating both a brand-new original song, *and* a beatmap to go along with it. Inspired by [BMS OF FIGHTERS](https://www.bmsoffighters.net/), the contest [kicked off starting with registrations in April](https://osu.ppy.sh/home/news/2024-05-10-kyoku-original-song-and-beatmapping-contest) and teams have spent the last **3 months** crafting their own works of art, all for a chance for become the top team and win both a prestigious badge, along with highly sought-after [contest points](/wiki/Contests/Contest_points).
+
+With 33 total entries the competition was fierce, and the judges had a lot of work ahead of them. After much deliberation, the dust has settled, and we are proud to present the top 3 entries.
+
+## Winners
+
+### 3rd place - Team RGB
+
+Drum and bass enjoyers can rejoice as Team RGB harnessed the full power of neurofunk. Featuring music by [Artackni](https://artackni.newgrounds.com/) and craftsmanship of [Amon-](https://osu.ppy.sh/users/18161041) and ballsmaster69, [*Encaged*](https://osu.ppy.sh/beatmapsets/2242804#osu/4767104) is a slider-heavy tech map that is bound to get the adrenaline pumping and keep your attention throughout its course.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://assets.ppy.sh/media/news/kyoku-2024-3rd-place.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Congratulations to Team RGB as they scored themselves 3rd place with a final score of **84.50**!
+
+### 2nd place - Zero Blunders
+
+Team Zero Blunders has put their best foot forward in this contest with a genre mash-up song featuring many osu! references. [*who's afraid?*](https://osu.ppy.sh/beatmapsets/2242420#osu/4766280) by fur:trash (a.k.a. [Nihkee](https://osu.ppy.sh/users/14244740) on osu!) offers many chances for [Mysty](https://osu.ppy.sh/users/10210657), [Le Mirai](https://osu.ppy.sh/users/13646997), [hakashii](https://osu.ppy.sh/users/11688893) and [[-Evil-]](https://osu.ppy.sh/users/10234313) to express their mapping ideas via music genres such as dariacore, dubstep and hyperpop, creating a balanced map that is both fun to look at and play.
```

Comment:
My name (Omekyu, https://osu.ppy.sh/users/14348073) is missing here, even though I was part of Team Zero Blunders as well. 
https://osu.ppy.sh/wiki/en/Contests/Kyoku/2024
![image](https://github.com/user-attachments/assets/2a94e0f4-07f9-4863-a6b5-78d040c3f2c8)

Reply:
@Walavouchey are you gonna be issuing updates to the wiki page? ill amend the news post now

Reply:
they're already in the wiki page